css/page/demos/page-example/index.html

```diff
@@ -0,0 +1,310 @@
+      color: #181A1B;
+    }
+
+    button.active {
+      background-color: #41E847;
+    }
+
+    button:hover {
+      background-color: white;
+    }
+
+    .code {
+      grid-area: pre;
+    }
+
+    pre {
+      font-family: "Roboto Mono";
+      white-space: pre-wrap;
+    }
+  </style>
+</head>
+
+<body>
+  <div class="container">
+    <div class="corner corner_1"></div>
+    <div class="before">
+      <div>←</div>
+      <p><span class="size-count"></span>&nbsp;мм</p>
+      <div>→</div>
+    </div>
+    <div class="left">
+      <div>↑</div>
+      <p class="size-text"><span class="size-count"></span>&nbsp;мм</p>
+      <div>↓</div>
+    </div>
+    <section class="page">
+      <h1>Я пример страницы</h1>
+      <p>
+        Это текст на странице
+      </p>
+    </section>
+    <div class="corner corner_2"></div>
+    <fieldset class="size">
+      <legend>Укажите размер бумаги</legend>
+      <div class="form">
+        <button class="makeA4 active">A4</button>
+        <button class="makeA5">A5</button>
+        <button class="makeLetter">Letter</button>
+        <button class="makeCustom">220 Х 300 мм</button>
+      </div>
+    </fieldset>
+    <fieldset class="orientation">
+      <legend>Укажите ориентацию страницы</legend>
+      <div class="form">
+        <button class="makeVertical active">Вертикальная</button>
+        <button class="makeHorizontal">Горизонтальная</button>
+      </div>
+    </fieldset>
+    <div class="code">
+      <pre>
+
+        @page {
+          sise: A4 vertical;
+        }
+      </pre>
+    </div>
+  </div>
+
+  <script>
+    let currentOrientation = 'vertical'
+    let currentSize = 'A4'
+
+    const container = document.querySelector('.container')
+    const sizeForm = container.querySelector('.size')
+    const a4Btn = sizeForm.querySelector('.makeA4')
+    const a5Btn = sizeForm.querySelector('.makeA5')
+    const letterBtn = sizeForm.querySelector('.makeLetter')
+    const customBtn = sizeForm.querySelector('.makeCustom')
+    const orientationForm = container.querySelector('.orientation')
+    const verticalBtn = orientationForm.querySelector('.makeVertical')
+    const horizontalBtn = orientationForm.querySelector('.makeHorizontal')
+    const sizeBtns = sizeForm.querySelectorAll('button')
+    const orientationBtns = orientationForm.querySelectorAll('button')
+    const pre = container.querySelector('pre')
+
+    const sizes = {
+      a4: {
+        width: 210,
+        height: 297
+      },
+      a5: {
+        width: 148,
+        height: 210
+      },
+      letter: {
+        width: 216,
+        height: 280,
+      },
+      custom: {
+        width: 220,
+        height: 300
+      }
+    }
+
+
+    function makeClasses(arr, item) {
+      arr.forEach((el) => {
+        el.classList.remove('active');
+      })
+      item.classList.add('active')
+    }
+
+    function makeCode(size, orientation) {
+      pre.textContent = `
+      @page {
+        size: ${size} ${orientation}
```

Comment:
Вот тут точка с запятой пропала и очень расстроилась

```suggestion
        size: ${size} ${orientation};
```